### PR TITLE
Remove second argument from some playSystemSound() calls in QML.

### PR DESCRIPTION
**Pull request by Penguin-Guru**
I don't know how this code was working before or why the position was needed, but the comments suggest that removing the position argument was the initial intention and the output test seems to work fine for me without it. May or may not fix https://github.com/vircadia/vircadia/issues/1453.

### CODE REVIEW

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-PR1473-597df36-_65dcd583.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-PR1473-597df36-.tar_65dcd583.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-PR1473-597df36-.tar_65dcd583.Z

**build (macOS-10.15, client)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-Interface-PR1473-597df36-_65dcd583.dmg

**build (macOS-10.15, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-PR1473-597df36-_65dcd583.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-PR1473-597df36-_65dcd583.exe

**build (self-hosted_debian-11_aarch64, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-PR1473-597df36-_65dcd583.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-PR1473-597df36-.tar_65dcd583.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-PR1473-597df36-.tar_65dcd583.Z

**Comment by ctrlaltdavid** [comment]
The second, position parameter was removed from `playSystemSound()` in commit `fa08e42`.

In addition to the changes made. the `qmlPosition` property should be marked as "deprecated" in the JSDoc in MyAvatar.h.

**Comment by Penguin-Guru** [comment]
Is that good?

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-10.15, client)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-Interface-PR1473-7afe1d8-_78981307.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-PR1473-7afe1d8-_78981307.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-PR1473-7afe1d8-.tar_78981307.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-PR1473-7afe1d8-.tar_78981307.Z

**build (macOS-10.15, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-PR1473-7afe1d8-_78981307.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-PR1473-7afe1d8-_78981307.exe

**build (self-hosted_debian-11_aarch64, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-PR1473-7afe1d8-_78981307.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-PR1473-7afe1d8-.tar_78981307.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-PR1473-7afe1d8-.tar_78981307.Z

File: interface/src/avatar/MyAvatar.h [hunk 1]
```diff
@@ -170,7 +170,7 @@ class MyAvatar : public Avatar {
      * @property {Vec3} skeletonOffset - Can be used to apply a translation offset between the avatar's position and the
      *     registration point of the 3D model.
      *
-     * @property {Vec3} qmlPosition - A synonym for <code>position</code> for use by QML.
+     * @property {Vec3} qmlPosition - A synonym for <code>position</code> for use by QML. This is deprecated.
```
Comment (ctrlaltdavid) [inline interface/src/avatar/MyAvatar.h:173]:
```suggestion
     * @property {Vec3} qmlPosition - A synonym for <code>position</code> for use by QML.
     *     <p class="important">Deprecated: This property is deprecated and will be removed.</p>
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]
Looking good. Just a doc change to make.

**Comment by ctrlaltdavid** [comment]
QA: Check that the "TEST YOUR SOUND" button works in Settings > Audio... dialog, both desktop and VR tabs.

**Review by ctrlaltdavid — APPROVED** [review]

**Comment by AleziaKurdis** [comment]
Tested with the Windows build: https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1473/Vircadia-PR1473-7afe1d8-_78981307.exe

Covered:
Audio app: "Test your Sound" button.

I have not too much idea of what is the Simplify UI. so that part has not been tested. (but according the code change... It would be surprising that this wouldn't work too.)

**Pull request merged**